docs/basics/stream_nwb.ipynb

```diff
@@ -36,7 +36,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    ### not all of these exist for all NWB files (Key Errors will arise if the fields don't exist for this file)
</pre><p>same as above review; can remove comment on line 2</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>